PROVISIONING-METAL.md

```diff
@@ -0,0 +1,169 @@
+The most immediate use of this feature for most users is setting `console` settings so boot messages can be seen on the appropriate consoles.
+
+In order to make use of this feature, one must create an initrd with the desired settings encoded inside it.
+(Bottlerocket technically doesn't use an initrd, so the file is empty save for the encoded boot config data)
+To create the initrd, you must first create a configuration file containing key value pairs for the settings you would like to pass to kernel / init.
+Full syntax is described in the [Boot Config documentation](https://www.kernel.org/doc/html/latest/admin-guide/bootconfig.html#config-file-syntax), but a simple example is provided below that shows the format of console settings as well as an example `systemd` parameter.
+
+The two acceptable prefixes to settings are `kernel` and `init`.
+Settings prefixed with `kernel` are added to the beginning of the kernel command line.
+Settings prefixed with `init` are added to the kernel command line after the `--`, but before any existing init parameters.
+
+In the example below, two console devices are set up, and `systemd`'s log level is set to `debug`.
+It is important to note that the kernel will output to both console devices, but `systemd` outputs to the last device in line (`ttyS1,115200n8` in the example below).
+
+Example Boot Configuration:
+```
+kernel {
+    console = tty0, "ttyS1,115200n8"
+}
+init {
+    systemd.log_level = debug
+}
+```
+
+Bottlerocket provides a command to create the Boot Configuration initrd from your config file, but there are a few prerequisites.
+You will need the [Bottlerocket code repo](https://github.com/bottlerocket-os/bottlerocket), Rust (via [rustup](https://rustup.rs/) or the official site), Docker (20.10.10 or later), and [cargo-make](https://sagiegurari.github.io/cargo-make/) (`cargo install cargo-make`).
+To build the Boot Configuration initrd, create a file called `bootconfig-input` in the root of the Bottlerocket repo containing your desired key/value pair kernel and init arguments.
+Then run:
+```
+cargo make boot-config
+```
+
+The above command will create the properly named initrd `bootconfig.data` in the root of the Bottlerocket repo.
+This is the file you will write to disk during provisioning.
+
+You can list a `bootconfig.data`'s contents, which also validates its format, by running:
+```
+cargo make validate-boot-config
+```
+
+## High level provisioning steps
+
+The high level steps to provision Bottlerocket to your host are below.
+Most provisioning systems provide methods to achieve the following:
+
+* Decompress (`unlz4`) and write the Bottlerocket image to the desired disk
+* Mount the `BOTTLEROCKET-PRIVATE` partition (partition 12)
+* Write the below files to the mounted partition:
+  * (Required) `user-data.toml`
+  * (Required) `net.toml`
+  * (Optional, recommended) `bootconfig.data`
+* Reboot
```

Comment:
This might be better at the top, before prerequisites, so the reader has the big picture before getting the details.